samples/exemplar/src/main/kotlin/com/squareup/exemplar/actions/DownloadAFileWebAction.kt

```diff
@@ -0,0 +1,30 @@
+package com.squareup.exemplar.actions
+
+import com.google.common.net.HttpHeaders
+import misk.security.authz.Unauthenticated
+import misk.tokens.TokenGenerator
+import misk.web.Get
+import misk.web.PathParam
+import misk.web.Response
+import misk.web.ResponseContentType
+import misk.web.actions.WebAction
+import misk.web.mediatype.MediaTypes
+import okhttp3.Headers.Companion.toHeaders
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class DownloadAFileWebAction @Inject constructor() : WebAction {
```

Comment:
Note: we don't have to keep this action in the exemplar if we don't want, it's just how I was manually testing this change

Reply:
I like the idea of continuing to add to the exemplar so it has fuller examples with the mutual benefit of end-to-end testing and local development for the dashboard.